rasa/nlu/model.py

```diff
@@ -289,11 +289,10 @@ def ensure_model_compatibility(
         model_version = metadata.get("rasa_version", "0.0.0")
         if version.parse(model_version) < version.parse(version_to_check):
             raise UnsupportedModelError(
-                "The model version is too old to be "
-                "loaded by this Rasa NLU instance. "
-                "Either retrain the model, or run with "
-                "an older version. "
-                "Model version: {} Instance version: {}"
+                "The model version is trained using Rasa {}"
```

Comment:
also note the addtional space at the end
```suggestion
                "The model version is trained using Rasa Open Source {} "
```